ui/raidboss/data/07-dt/ultimate/futures_rewritten.txt

```diff
@@ -65,74 +65,74 @@ hideall "--sync--"
-349.4 "Powerful Light" Ability { id: "9D19", source: "Usurper of Frost" }
-358.4 "Bright Hunger" Ability { id: "9D15", source: "Usurper of Frost" }
-361.5 "Banish III" Ability { id: ["9D1C", "9D1D"], source: "Usurper of Frost" }
-370.4 "The House of Light" Ability { id: "9CFC", source: "Usurper of Frost" }
-375.8 "--sync--" Ability { id: "9CEF", source: "Usurper of Frost" }
-379.1 "--sync--" StartsUsing { id: "9D20", source: "Usurper of Frost" }
-389.7 "Absolute Zero (enrage)" Ability { id: "9D8E", source: "Usurper of Frost" }
-
 # Adds Phase
-392.0 "Swelling Frost" Ability { id: "9D21", source: "Usurper of Frost" }
-424.0 "Sinbound Blizzard III" Ability { id: "9D42", source: "Crystal of Darkness" }
-425.0 "Hiemal Storm" Ability { id: "9D40", source: "Crystal of Light" }
-428.2 "Hiemal Storm" Ability { id: "9D40", source: "Crystal of Light" }
-429.2 "Sinbound Blizzard III" Ability { id: "9D42", source: "Crystal of Darkness" }
-431.4 "Hiemal Storm" Ability { id: "9D40", source: "Crystal of Light" }
-434.3 "Sinbound Blizzard III" Ability { id: "9D42", source: "Crystal of Darkness" }
-434.5 "Hiemal Storm" Ability { id: "9D40", source: "Crystal of Light" }
-437.7 "Hiemal Storm" Ability { id: "9D40", source: "Crystal of Light" }
-439.5 "Sinbound Blizzard III" Ability { id: "9D42", source: "Crystal of Darkness" }
-440.9 "Hiemal Storm" Ability { id: "9D40", source: "Crystal of Light" }
-444.1 "Hiemal Storm" Ability { id: "9D40", source: "Crystal of Light" }
-444.8 "Sinbound Blizzard III" Ability { id: "9D42", source: "Crystal of Darkness" }
-447.3 "Hiemal Storm" Ability { id: "9D40", source: "Crystal of Light" }
-450.1 "Sinbound Blizzard III" Ability { id: "9D42", source: "Crystal of Darkness" }
+392.4 "Swelling Frost" Ability { id: "9D21", source: "Usurper of Frost" }
+424.4 "Sinbound Blizzard III" Ability { id: "9D42", source: "Crystal of Darkness" }
+425.4 "Hiemal Storm" Ability { id: "9D40", source: "Crystal of Light" }
+428.6 "Hiemal Storm" Ability { id: "9D40", source: "Crystal of Light" }
+429.6 "Sinbound Blizzard III" Ability { id: "9D42", source: "Crystal of Darkness" }
+431.8 "Hiemal Storm" Ability { id: "9D40", source: "Crystal of Light" }
+434.7 "Sinbound Blizzard III" Ability { id: "9D42", source: "Crystal of Darkness" }
+434.9 "Hiemal Storm" Ability { id: "9D40", source: "Crystal of Light" }
+438.1 "Hiemal Storm" Ability { id: "9D40", source: "Crystal of Light" }
+439.9 "Sinbound Blizzard III" Ability { id: "9D42", source: "Crystal of Darkness" }
+441.3 "Hiemal Storm" Ability { id: "9D40", source: "Crystal of Light" }
+444.5 "Hiemal Storm" Ability { id: "9D40", source: "Crystal of Light" }
+445.2 "Sinbound Blizzard III" Ability { id: "9D42", source: "Crystal of Darkness" }
+447.7 "Hiemal Storm" Ability { id: "9D40", source: "Crystal of Light" }
+450.5 "Sinbound Blizzard III" Ability { id: "9D42", source: "Crystal of Darkness" }
 
 # Phase Three
 500.0 "Junction" Ability { id: "9D22", source: "Usurper of Frost" } window 500.0,0
```

Comment:
If you want to do a pre-sync here before Junction so it will actually show up correctly on the timeline in advance, you could sync to the `WasDefeated` line for the Ice Veil, e.g.:
`25|[ts]|40008D78|Ice Veil|40008D53|`

It happens 11.2s before the Junction ability lines, so `488.8`.

Reply:
That's fine, I'll add that. I'd prefer not to touch more on the intermission than that, since I was focused specifically on the main P2 section here, and my group still hasn't got past Ultimate Relativity.